### PR TITLE
Task mailbox clean

### PR DESCRIPTION
**Pull request by beevans**
Filesync work

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2000)

### CODE REVIEW

File: iml-agent/src/action_plugins/stratagem/action_purge.rs [hunk 1]
```diff
@@ -54,7 +54,7 @@ pub async fn read_mailbox(
         .try_filter_map(future::ok)
         .chunks(rmfids_size)
         .map(|xs| xs.into_iter().collect())
-        .try_for_each_concurrent(10, |fids| {
+        .try_for_each(|fids| {
```
Comment (utopiabound) [inline iml-agent/src/action_plugins/stratagem/action_purge.rs:57]: Why?

File: chroma_core/models/client_mount.py [hunk 2]
```diff
@@ -298,17 +302,16 @@ def description(self):
...
                 (
                     ObjectCache.get_one(ManagedFilesystem, lambda mf, mtd=m: mf.name == mtd.filesystem).mount_path(),
-                    m.mountpoint if m.mountpoint else "/mnt/{}".format(m.filesystem),
+                    m.mountpoint,
                 )
                 for m in unmounted
             ],
-        }
-
+        )
```
Comment (utopiabound) [inline chroma_core/models/client_mount.py:314]: this looks like it's rolling back changes...

File: iml-agent/src/action_plugins/stratagem/action_purge.rs (hunk 1 above)
Comment (beevans) [reply iml-agent/src/action_plugins/stratagem/action_purge.rs:57]: Should have been in action_filesync.rs, and I didn't back it out

**Comment by jgrund** [comment]
Is this patch still in flight? It appears to be a bit stale behind master.

**Comment by beevans** [comment]
No, 2007 should be the correct PR.  I'll close this.

**Pull request closed without merging**